build.sbt

```diff
@@ -31,33 +31,39 @@ lazy val backend = (project in file("backend"))
   .dependsOn(services)
   .dependsOn(core % "test->test")
 
+lazy val sfsBackend = (project in backendRoot / "sfs")
```

Comment:
I'm wondering if we could pick a name for this project that makes it clear this is an abstract supertype for building shared filesystem backends, without wrongly implying it is itself a usable backend.  `sfsBackend` and`lsfBackend` look so similar but represent such different ideas... 😄